assets/source/setup-guide/app/views/WizardApp.js

```diff
@@ -29,14 +25,16 @@ import {
 /**
  * Onboarding Wizard component.
  *
+ * @param {Object} props React props.
+ * @param {Object} [props.query] The current query string, parsed into an object, from history.
```

Comment:
it'd be nicer to specify `query` is not an optional prop and comes from page URL.

```suggestion
 * @param {Object} props.query The current query string, parsed into an object, from the page URL.
```

Reply:
Perfect here is the JSDoc updated: [8bb0e7](https://github.com/woocommerce/pinterest-for-woocommerce/pull/318/commits/8bb0e7fe820186b2bf2e48198ddd667ac9462cc9)